app/config.py

```diff
@@ -13,6 +13,7 @@
 DEFAULT_ERROR_AZURE_FAILED = "Azure index search failed."
 DEFAULT_ERROR_FINESSE_DATA_FAILED = "finesse-data static search failed"
 DEFAULT_ERROR_UNEXPECTED = "Unexpected error."
+DEFAULT_FUZZY_MATCH_THRESHOLD = "90"
```

Comment:
why not make this a number instead of a string? int casting still works on something that is already an int.
```
Python 3.10.8 (main, Nov  2 2023, 18:07:42) [GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> int(90)
90
```

#

Reply:
Done.